client/tests/integration/smartcontracts/validator_with_migration_fail/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "validator_with_migration_fail"
+
+edition.workspace = true
```

Comment:
Wow, I haven't known such a syntax is allowed in TOML.

Reply:
One of the main reasons why even Python is starting to use TOML in their package configurations.